core/domain/prod_validation_jobs_one_off.py

```diff
@@ -732,7 +732,11 @@ def map(model_instance):
 
     @staticmethod
     def reduce(key, values):
-        if len(values) > 1:
+        # If normalized name is not set, we do not comapare it with normalized
+        # names for other users. It is not mandatory to set the normalized
+        # user names in UserSettingsModel since the gae_id is used as
```

Comment:
I'm not sure this explanation is correct. I think the actual explanation is that some users who have logged in but not completed the sign-up process may not have a username specified yet.

Reply:
Ah ok, wasn't aware of this. Updated.